tests/ui/stable-mir-print/basic_function.stdout

```diff
@@ -0,0 +1,234 @@
+// WARNING: This is highly experimental output it's intended for stable-mir developers only.
+// If you find a bug or want to improve the output open a issue at https://github.com/rust-lang/project-stable-mir.
+fn foo(_0: i32) -> i32 {
+    let mut _0: (i32, bool);
+}
+    bb0: {
+        _2 = 1 Add const 1_i32
+        assert(!move _2 bool),"attempt to compute `{} + {}`, which would overflow", 1, const 1_i32) -> [success: bb1, unwind continue]
+    }
+    bb1: {
+        _0 = move _2
+        return
+    }
+fn bar(_0: &mut Ty {
+    id: 10,
+    kind: RigidTy(
+        Adt(
```

Comment:
Not for this PR, but we probably don't want to print the structure itself, but what it represents.

Reply:
Actually I just noticed that you fixed that in #118364. :smile:

Reply:
Yeah, I wanted this PR just to print the status quo, and subsequent PRs could actually do the real output fixing.